env.example

```diff
@@ -53,3 +53,6 @@ NODE_TLS_REJECT_UNAUTHORIZED=0
 languageWorkers__node__arguments="--inspect=0.0.0.0:5861"
 
 # AzureWebJobs.UpdateVisibleServicesCache.Disabled=true
+
+QueueStorageConnection=<STORAGE_CONNECTION_STRING>
```

Comment:
can we rename `QueueStorageConnection` to `SubscriptionFeedStorageConnection` ?

Reply:
Of course, just note that it's the same value used here https://github.com/pagopa/io-functions-app/blob/master/UpdateSubscriptionsFeedActivity/index.ts#L18